src/amount.cpp

```diff
@@ -12,14 +12,14 @@ const std::string CURRENCY_UNIT = "BTC";
 CFeeRate::CFeeRate(const CAmount& nFeePaid, size_t nSize)
 {
     if (nSize > 0)
-        nSatoshisPerK = nFeePaid*1000/nSize;
+        nSatoshisPerK = nFeePaid * PRECISION_MULTIPLIER / nSize;
```

Comment:
Yeah, see #7731 . GetFee is over complicated there, but if you assert ```assert(PRECISION_MULTIPLIER % KB == 0)``` that code can be simplified. I mean, it can be simplified by more means, for example, removing the special case from https://github.com/bitcoin/bitcoin/pull/7705#discussion_r56851706 (although that will be greatly disruptive outside of the class).